cocos/tween/tween.ts

```diff
@@ -100,7 +104,7 @@ export class Tween<T> {
      * @method target
      * @param target @en The target of this tween @zh 当前缓动的目标对象
      */
-    target (target: T): Tween<T | undefined> {
+    target (target: T): Tween<T> {
```

Comment:
`this` is `Tween<T>` type, I don't know why it returned `Tween<T | undefined>` before.